### PR TITLE
Add debian installer.

### PR DESCRIPTION
**Pull request by MalikMlitat**
solves issue #8

### CODE REVIEW

**Comment by MalikMlitat** [comment]
hier how its look like:
[debian_pkg.tar.gz](https://github.com/ShibeTechnology/dogecoin-spv-node/files/9280478/debian_pkg.tar.gz)

File: Makefile [hunk 1]
```diff
@@ -18,14 +18,29 @@ clean-regtest:
...
+	rm -rf dist/debian/
+	mkdir -p dist/debian/dogecoin-spv/DEBIAN/;mkdir -p dist/debian/dogecoin-spv/usr/bin/
+	cp -r dist/linux/* dist/debian/dogecoin-spv/usr/bin/
+	# create control file
+	touch $(CONTROL_FILE)
+	echo "Package: dogecoin-spv" >> $(CONTROL_FILE)
+	echo "Architecture: amd64" >> $(CONTROL_FILE)
+	echo "Version: $(VERSION)-1" >> $(CONTROL_FILE)
+	echo "Maintainer: Malik <malik.mlitat@gmail.com>" >> $(CONTROL_FILE)
+	echo "Description: simple spv node for Dogecoin." >> $(CONTROL_FILE)
```
Comment (rllola) [inline Makefile:40]: We actually have now a spv node + wallet. So in the description we could say "A simple spv wallet for Dogecoin."

**Review by rllola — CHANGES_REQUESTED** [review]

**Comment by rllola** [comment]
I tested it on the CI on my repo. And I got an error
```
Error: ENOENT: no such file or directory, stat './dogecoin-spv_0.5.15.deb'
```

**Comment by MalikMlitat** [comment]
I'll check it today, could you send me a link where you tested it? I done that part blindly, since i didnt have any CI

File: Makefile (hunk 1 above)
Comment (MalikMlitat) [reply Makefile]: fixed

**Comment by MalikMlitat** [comment]
@rllola see test release :) 
 https://github.com/MalikMlitat/dogecoin-spv-node/releases/tag/untagged-7f16b179e814c1d73d0f

**Review by rllola — APPROVED** [review]

**Comment by rllola** [comment]
Merged! Thank you @MalikMlitat

**Pull request merged**